modules/class_module.js

```diff
@@ -0,0 +1,74 @@
+import getDateAndTime from './date_and_time.js';
+
+const bookListContainer = document.querySelector('.book-list');
+const form = document.querySelector('form');
+const titleInput = document.querySelector('.title');
+const authorInput = document.querySelector('.author');
+const datePara = document.querySelector('.date--para');
+class BookList {
+  constructor() {
+    this.bookArray = [];
+  }
+
+  addBook(title, author, id) {
+    const newBook = { title, author, id };
+    this.bookArray = [...this.bookArray, newBook];
+  }
+
+  removeBook(id) {
+    this.bookArray = this.bookArray.filter((book) => book.id !== id);
+  }
+
+  displayBook() {
+    datePara.textContent = getDateAndTime();
+    bookListContainer.innerHTML = '';
+    const bookElement = this.bookArray.map((book) => {
+      const bookCard = `  <li>
```

Comment:
Please kindly refactor all these to arrow functions as stated in the requirement. 👍 

<img width="337" alt="Untitled" src="https://github.com/Akorede3133/Awesome_Books_With_ES6/assets/103386183/771b416e-2047-43c4-bd99-3f63c4fdf032">